src/tests/ftest/control/ms_resilience.py

```diff
@@ -0,0 +1,314 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2021 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+"""
+from __future__ import print_function
+
+from apricot import TestWithServers
+from general_utils import stop_processes
+import random
+import socket
+import time
+
+
+class ManagementServiceResilience(TestWithServers):
+    """Test Class Description:
+    Verify that MS is accessible as long as one instance is left.
+    :avocado: recursive
+    """
+
+    def __init__(self, *args, **kwargs):
+        """Inititialize a ManagementServiceResilience object."""
+        super(ManagementServiceResilience, self).__init__(*args, **kwargs)
```

Comment:
We're using python3 now:
```suggestion
        super().__init__(*args, **kwargs)
```

Reply:
Fixed